hibernate-core/src/main/java/org/hibernate/dialect/pagination/InformixLimitHandler.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.dialect.pagination;
+
+import java.util.Locale;
+
+import org.hibernate.engine.spi.RowSelection;
+
+public class InformixLimitHandler extends AbstractLimitHandler {

```

Comment:
I'd recommend we name this as `Informix10LimitHandler` to be consistent.

I also recommend given that this limit handler implementation maintains no state, rather than creating a new instance everytime the dialect's `#getLimitHander()` is invoked, we add a public static variable and a private constructor here and use it in the dialect's getter method, as shown here:

<pre>
public class Informix10LimitHandler extends AbstractLimitHandler {
  public static Informix10LimitHandler INSTANCE = new InformixLimitHandler();

  private Informix10LimitHandler() {
    // Disallow instantiation
  }
  
  // rest of your class
}
</pre>

Then in your `Informix10Dialect`:
<pre>
@Override
public LimitHandler getLimitHandler() {
  return Informix10LimitHandler.INSTANCE;
}
</pre>